python/aitemplate/backend/rocm/normalization/layernorm.py

```diff
@@ -65,34 +65,45 @@
 EXEC_TEMPLATE = jinja2.Template(
     """
     std::vector<ck::index_t> i_inStrides;
-
+    std::vector<ck::index_t> i_outStrides;
+    {% if input_strides is defined %}
+    i_inStrides.push_back({{input_strides[-2]}});
+    i_inStrides.push_back({{input_strides[-1]}});
+    {% else %}
     i_inStrides.push_back(N);
     i_inStrides.push_back(1);
+    {% endif %}
 
+    {% if output_strides is defined %}
+    i_outStrides.push_back({{output_strides[-2]}});
+    i_outStrides.push_back({{output_strides[-1]}});
+    {% else %}
+    i_outStrides.push_back(N);
+    i_outStrides.push_back(1);
+    {% endif %}
 
     auto device_instance = {{instance}}{};
     auto argument_ptr = device_instance.MakeArgumentPointer(
         {M, N},
         i_inStrides,
         std::vector<ck::index_t>{0, 1},
         std::vector<ck::index_t>{0, 1},
-        i_inStrides,
+        i_outStrides,
         {1},
         {{eps}},
-        static_cast<ck::half_t *>(input),
+        static_cast<ck::half_t *>(input) + {{ input_offset if input_offset is defined else 0 }},
         static_cast<ck::half_t *>(gamma),
         static_cast<ck::half_t *>(beta),
-        static_cast<ck::half_t *>(output),
+        static_cast<ck::half_t *>(output) + {{ output_offset if output_offset is defined else 0 }},
```

Comment:
Same here - ```output_offset``` is defined and its default value is ```0```, because ```output_offset``` comes from ```func_attrs["output_accessors"][0].offset```, which is defined at:

https://github.com/facebookincubator/AITemplate/blob/main/python/aitemplate/compiler/tensor_accessor.py#L45

Reply:
No, template may not render with the output_offset argument, so I have to check it.

Reply:
I see. It probably comes from ```norm_common.gen_profiler```. Thanks.